CI-2: fleet-dispatch.yml SHA-pinned action lacks version comment

## Problem

CI-2 diagnostics currently fail on `main`-baseline content when running the full test suite for unrelated PRs. The failing assertion is from `tests/kb/test_workflow_sha_pins.py::WorkflowShaPinTests::test_pinned_refs_have_version_comments`.

Observed on PR #333 / run `27895948775`, job `82547385568`:

```text
AssertionError: Regex didn't match: '@[0-9a-f]{40}\\s+#' not found in '        uses: actions/create-github-app-token@bcd2ba49218906704ab6c1aa796996da409d3eb1' : fleet-dispatch.yml:151 — SHA-pinned ref missing version comment: 'uses: actions/create-github-app-token@bcd2ba49218906704ab6c1aa796996da409d3eb1'
```

## Expected remediation

Add the required version comment to the SHA-pinned `actions/create-github-app-token` reference in `fleet-dispatch.yml`, or update the contract test if the expected format has changed.

## Validation

- `python3 -m pytest tests/kb/test_workflow_sha_pins.py -v`
- CI-2 diagnostics should pass this pytest gate.

## Context

Found while implementing unrelated issue #303. Not fixed there to keep that PR scoped to `extract_frontmatter` tests/docstring only.

--- a/.github/workflows/fleet-dispatch.yml
+++ b/.github/workflows/fleet-dispatch.yml
@@ -148,7 +148,7 @@ jobs:
       - name: Create GitHub App token for auto-merge
         id: app-token
         if: steps.app-token-inputs.outputs.available == 'true'
-        uses: actions/create-github-app-token@bcd2ba49218906704ab6c1aa796996da409d3eb1
+        uses: actions/create-github-app-token@bcd2ba49218906704ab6c1aa796996da409d3eb1 # v3.2.0
         continue-on-error: true
         with:
           app-id: ${{ secrets.GH_APP_ID }}

